frontend/src/components/InstallPrompt.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button } from '@/components/ui/button'
+import { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+// Define the type for the beforeinstallprompt event
+interface BeforeInstallPromptEvent extends Event {
+  platforms: string[]
+  userChoice: Promise<{ outcome: 'accepted' | 'dismissed'; platform: string }>
+  prompt(): Promise<void>
+}
+
+const InstallPrompt = () => {
```

Comment:
This shouldn't be visible on desktop screens.
Also, it should not re-appear after closing it, so store it in localstorage to make sure it doesn't keep popping up?